_includes/components/create-an-account.html

```diff
@@ -19,9 +19,7 @@
           </div>
         </div>
         <div class="sidebar desktop:grid-col-4 desktop:grid-offset-1 desktop-lg:grid-col-3 desktop-lg:grid-offset-2">
-          <div class="box">
-            {{ include.info | markdownify }}
-          </div>
+          {% include alert.html content=include.info %}
```

Comment:
looks like there are a handful of places we use a the usa-alert, should we file a follow-up ticket to standardize these?
```
> git grep -l usa-alert | grep -v .scss | grep -v \\.ts
_includes/contact_form.html
_includes/country_support_table.html
_layouts/contact_us.html
content/_help/trouble-signing-in/issues-with-authentication-methods._en.md
content/_help/trouble-signing-in/issues-with-authentication-methods._es.md
content/_help/trouble-signing-in/issues-with-authentication-methods._fr.md
```

Reply:
It'll take me less time to create a quick follow-on pull request than to write a ticket, so I'll commit to that 😄